miden-crypto/src/merkle/mmr/partial.rs

```diff
@@ -159,7 +159,7 @@ impl PartialMmr {
     /// # Errors
     /// Returns an error if the specified position is greater-or-equal than the number of leaves
     /// in the underlying MMR.
-    pub fn open(&self, pos: usize) -> Result<Option<MmrProof>, MmrError> {
+    pub fn open(&self, pos: usize) -> Result<Option<MmrPath>, MmrError> {
```

Comment:
This is fine for now, but I wonder if we should make it more consistent and return `MmProof`. This would require a somewhat deeper refactoring of the `PartialMmr` - but maybe it is worth it.

Let's create an issue for this.